### PR TITLE
Fixes in getting started content

### PR DESCRIPTION
**Pull request by blackdwarf**
- Fix for typo on OS X starting instructions
- Fix for Windows PS starting instructions

Fix (somewhat) #8

### CODE REVIEW

File: _includes/core/windows_gs.html [hunk 1]
```diff
@@ -2,11 +2,11 @@
     <div class="step step-none windows-trail" id="step-1">
       <div class="step-number">1</div>
       <h3>Install .NET Version Manager (DNVM)</h3>
-      <p>DNVM is a script that will help us manage execution environments that we will use. Installing it is a breeze: </p>
+      <p>DNVM is a script that will help us manage execution environments that we will use. Installing it is a breeze using PowerShell: </p>
       <div class="terminal">
         <div class="terminal-titlebar"></div>
         <div class="terminal-body">
-          <p><span class="prompt windows-prompt"></span> @powershell -NoProfile -ExecutionPolicy unrestricted -Command "&{$Branch='dev';iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/aspnet/Home/dev/dnvminstall.ps1'))}"</p>
+          <p><span class="prompt windows-prompt"></span> "&{$Branch='dev';iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/aspnet/Home/dev/dnvminstall.ps1'))}"</p>
```
Comment (akoeplinger) [inline _includes/core/windows_gs.html]: Do the chars like `&` need to be escaped as this is raw HTML?

**Comment by akoeplinger** [comment]
LGTM other than that.

**Comment by thesmithee** [comment]
Thanks for the catch. Fixed it and will merge.

File: _includes/core/windows_gs.html [hunk 2]
```diff
@@ -2,11 +2,11 @@
     <div class="step step-none windows-trail" id="step-1">
       <div class="step-number">1</div>
       <h3>Install .NET Version Manager (DNVM)</h3>
-      <p>DNVM is a script that will help us manage execution environments that we will use. Installing it is a breeze: </p>
+      <p>DNVM is a script that will help us manage execution environments that we will use. Installing it is a breeze using PowerShell: </p>
       <div class="terminal">
         <div class="terminal-titlebar"></div>
         <div class="terminal-body">
-          <p><span class="prompt windows-prompt"></span> @powershell -NoProfile -ExecutionPolicy unrestricted -Command "&{$Branch='dev';iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/aspnet/Home/dev/dnvminstall.ps1'))}"</p>
+          <p><span class="prompt windows-prompt"></span> "&amp;{$Branch='dev';iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/aspnet/Home/dev/dnvminstall.ps1'))}"</p>
```
Comment (ghost) [inline _includes/core/windows_gs.html:9]: This is wrong, it should be without quotes:

``` html
<p><span class="prompt windows-prompt"></span> &amp;{$Branch='dev';iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/aspnet/Home/dev/dnvminstall.ps1'))}</p> 
```

=>

``` powershell
&{$Branch='dev';iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/aspnet/Home/dev/dnvminstall.ps1'))}
```

to execute it run from PS.

**Pull request merged**